File: Herencia/ejercicio 3/Java/Docente.java
class Docente extends Persona {
    public String nit, profesion, especialidad;
    public Docente(String ci, String nombre, String apellido, String celular, LocalDate fechaNac, String sexo,
                   String nit, String profesion, String especialidad) {
        super(ci, nombre, apellido, celular, fechaNac, sexo);
        this.nit = nit;
        this.profesion = profesion;
        this.especialidad = especialidad;
    }
    @Override
    public void mostrar() {
        super.mostrar();
        System.out.println("Profesión: " + profesion + ", Especialidad: " + especialidad);
    }
    public String getProfesion() {
        return profesion;
    }
}

File: Herencia/ejercicio 3/Python/Docente.py
from Persona import Persona
class Docente(Persona):
    def __init__(self, ci, nombre, apellido, celular, nacimiento, sexo, nit, profesion, especialidad):
        super().__init__(ci, nombre, apellido, celular, nacimiento, sexo)
        self.nit = nit
        self.profesion = profesion
        self.especialidad = especialidad
    def mostrar(self):
        super().mostrar()
        print(f"Profesion: {self.profesion}, Especialidad: {self.especialidad}")